iib/workers/tasks/build.py

```diff
@@ -671,10 +671,19 @@ def inspect_related_images(bundles: List[str], request_id) -> None:
     :raises IIBError: if one of the bundles does not have the pullable related_image.
     """
     invalid_related_images = []
+    replace_related_image_registry = None
```

Comment:
Is there a reason behind declaring it first here?

Reply:
I recall it was failing on one case, but let me check again

Reply:
Updated